app/_locales/en/messages.json

```
@@ -421,10 +421,10 @@
     "message": "Disconnect All"
   },
   "disconnectAllModalDescription": {
-    "message": "Are you sure? You will no longer be able to interact with any of these sites."
+    "message": "Are you sure? You will be disconnected from all sites on all accounts."
   },
   "disconnectAccountModalDescription": {
-    "message": "Are you sure? You will no longer be able to interact with this site."
+    "message": "Are you sure? Your account will be disconnected from this site."
```

I agree that the old wording sounded too "final". This is better. But it does seem a little.... redundant? The user had clicked on a "Disconnect Account" button prior to this message being shown after all.

Not blocking on this, but maybe adding the account name here would make the message more informative? e.g.

```suggestion
    "message": "Are you sure? Your account (\"$1\") will be disconnected from this site."
```